InsiteTeamTask/ClientApp/src/app/attendance/attendance.component.html

```diff
@@ -0,0 +1,82 @@
+<p>Select Attendence by Game and Season</p>
+
+<div class="row">
+  <div class="col-6">
+    <div class="row">
+      <div class="col-6">Season</div>
+      <div class="col-6">
+        <div *ngIf="seasons">
+          <select (change)="setSeason($event.target.value)">
```

Comment:
The use of the `(change)` event handler isn't necessary here - you can utilise Angular's [2-way binding with ngModel](https://angular.io/guide/two-way-binding) to make sure your selected season is always the same as this select element's value.